udata/auth/__init__.py

```diff
@@ -3,27 +3,28 @@
 
 import logging
 
-from flask import render_template, current_app, Blueprint, request, redirect
-from flask.ext.principal import (  # noqa
-    Permission as BasePermission, PermissionDenied, identity_loaded, RoleNeed,
-    UserNeed
-)
+from flask import current_app
+from flask import render_template
+
+from flask.ext.principal import identity_loaded
```

Comment:
In that case: 1. update the related documentation 2. update all imports for consistency ;)